src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -408,7 +408,12 @@ private void errorHandler(RoutingContext routingContext) {
         String message = null;
         // in case of validation exception, building a meaningful error message
         if (routingContext.failure() != null && routingContext.failure() instanceof ValidationException) {
-            ValidationException validationException = (ValidationException) routingContext.failure();
+            ValidationException validationException = new ValidationException("null");
+            try {
+                validationException = (ValidationException) routingContext.failure();
+            } catch (ClassCastException e) {
+                e.printStackTrace();
+            }
```

Comment:
Why?

Reply:
what damn does it mean, we have an `if` before that

`if (routingContext.failure() != null && routingContext.failure() instanceof ValidationException)`

which confirm that it's a `ValidationException`